.build/plugins/cache/Generate_Manual-state.json

```diff
@@ -0,0 +1,114 @@
+{
+  "commandLine" : [
+    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc",
+    "-L",
+    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/PluginAPI",
+    "-lPackagePlugin",
+    "-Xlinker",
+    "-rpath",
+    "-Xlinker",
+    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/PluginAPI",
+    "-target",
+    "arm64-apple-macosx11.0",
+    "-sdk",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk",
+    "-F",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks",
+    "-I",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/lib",
+    "-L",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/lib",
+    "-g",
+    "-swift-version",
+    "5",
+    "-package-description-version",
+    "5.6.0",
+    "-I",
+    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/PluginAPI",
+    "-sdk",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk",
+    "-parse-as-library",
+    "-Xfrontend",
+    "-serialize-diagnostics-path",
+    "-Xfrontend",
+    "/Users/marium.hassan/JustTweak/.build/plugins/cache/Generate_Manual.dia",
```

Comment:
looking at user-specific lines like this one, are we sure we want to commit the `.build` folder at all?

Reply:
+1, this folder should be git ignored